test/integration/uninstall_int_test.go

```diff
@@ -139,6 +139,11 @@ func (suite *UninstallIntegrationTestSuite) testUninstall(all bool) {
 		}
 	}
 
+	if runtime.GOOS == "windows" {
+		shortcutDir := filepath.Join(ts.Dirs.HomeDir, "AppData", "Roaming", "Microsoft", "Windows", "Start Menu", "Programs", "ActiveState")
```

Comment:
Are we verifying that this file exists after install? Otherwise this could be a false positive.